docs/data-sources/app.md

```diff
@@ -14,8 +14,7 @@ Returns the Fortanix DSM app object from the cluster as a Data Source.
 
 ```terraform
 data "dsm_app" "sample_app" {
-    app_id = "36cc21d0-a852-4c29-8e34-33388ab73529"
-    new_credential = false
+  app_id = "36cc21d0-a852-4c29-8e34-33388ab73529"
```

Comment:
instead have place holder for app_id, like <app_uuid>

Reply:
app_id is not part of my PR, it is from previous releases. Having said that, we had discussed and agreed in our previous meetings to include the full app_id string in the example to give users a clear idea of how the Terraform block should look. If you still think, we need a placeholder, I can include that in my PR.

Reply:
Generally we want to indicate placeholders. But if it was previously discussed to go with the current approach then it should be ok.